src/EventDispatcher/SimpleEventDispatcher.php

```diff
@@ -54,7 +56,30 @@ public function isInstanceOfTriggered(string $class, int $times = null): bool
         return $this->processBoolResult(static fn(object $event): bool => $event instanceof $class, $times);
     }
 
-    private function processBoolResult(Closure $closure, ?int $times)
+    public function expectsEvent(string $eventClass, int $times): void
```

Comment:
```suggestion
    public function expectEvent(string $eventClass, int $times): void
```

Reply:
This naming is convenient with [PhpUnit's mock](https://phpunit.readthedocs.io/en/latest/test-doubles.html#mock-objects) naming.